Python/Tests/ReplWindowUITests/ReplWindowPythonTests.cs

```diff
@@ -263,7 +265,7 @@ public abstract class ReplWindowPythonTests : ReplWindowPythonSmokeTests {
             // where we will re-enable them when they work properly.
             using (var interactive = Prepare()) {
                 int spaces = interactive.TextView.Options.GetOptionValue(DefaultOptions.IndentSizeOptionId);
-                int textWidth = interactive.Settings.PrimaryPrompt.Length + 3;
+                int textWidth = ">>>".Length + 3;
```

Comment:
Would rather not hardcode prompts anywhere, since the interactive window should be reporting what its prompts are. Maybe that won't work in tests? If it doesn't seem easy, leave a comment here about it.